runtime/CodeEmitter.cs

```diff
@@ -1842,7 +1852,11 @@ private void DumpMethod()
 
 		internal void DefineSymbolDocument(ModuleBuilder module, string url, Guid language, Guid languageVendor, Guid documentType)
 		{
+#if NETSTANDARD
+			throw new PlatformNotSupportedException();
+#else
 			symbols = module.DefineDocument(url, language, languageVendor, documentType);
```

Comment:
May be targeting .net standard 2.1 brings module builder API https://docs.microsoft.com/ru-ru/dotnet/api/system.reflection.emit.modulebuilder